packages/dapp/src/hooks/useBlockchainExplorerURL.ts

```diff
@@ -3,14 +3,12 @@ import { useWallet } from '../providers';
 
 export const useBlockchainExplorerURL = () => {
   const { account } = useWallet();
-  const { getChainById } = useChains();
+  const { isSuccess, getChainById } = useChains();
 
-  if (account?.address && account?.chainId) {
+  if (account.isActive && account.chainId) {
     const chain = getChainById(account.chainId);
-    if (chain?.metamask) {
+    if (isSuccess) {
       return `${chain.metamask.blockExplorerUrls[0]}address/${account.address}`;
-    } else {
-      console.error(`No blockchain explorer found for ${account.chainId}`);
```

Comment:
let's leave this error log here.
It's tends to be informative imo if something goes wrong.

The flow of execution was being interrupted due to `throw Error(...)` https://github.com/lifinance/jumper.exchange/blob/2c3148e060b233b01ff889cac4cfce75ebec9137/packages/dapp/src/hooks/useChains.ts#L30

Reply:
@Abhikumar98 Sounds legit, I added it back